core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -221,7 +232,9 @@ export class RteHelperModalComponent {
     });
 
     this.customizationArgsForm = this.fb.group(formGroupControls);
-
+    if (this.componentId === 'video') {
```

Comment:
Better to declare named constants that you reference -- COMPONENT_ID_VIDEO, COMPONENT_ID_LINK, etc.

Reply:
The latter is fine; just put the constants together near the top.

Reply:
@yungmulah Please do not resolve open comments. Reply to them instead. See guidelines for responding to comments [here](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve).